### ISSUE TITLE
[BUG] Factories keep saying not built

### ISSUE THREAD

**Issue by sarr95** [GabrielOlvH/Industrial-Revolution#384]
**Description:**
- i can not get the factories to work it keeps acting like it is not built

**Steps to Reproduce:**
- Steps to reproduce this behaviour

**Technical Information:**
 - Minecraft: 1.18.1
 - Fabric API: fabric-api-0.45.2+1.18
 - Industrial Revolution: indrev-1.13.6-BETA
 - Modpack: 

**Logs:**
- Use a paste service, do not paste the logs directly here.

**Details:**
Extra information that might be helpful like screenshots or technical information about your computer.

**Comment by GabrielOlvH** [comment]
Are you sure every block is facing the correct way? Send screenshots

**Comment by sarr95** [comment]
yes i am 100% everything is facing the right way me an my buddy tried to get it working but it only works above ground

**Comment by CataratasDolly** [comment]
Same thing here, tried building it in my base, at y level -50, did not work. Tried at y level 87 and it worked.
